chia/wallet/notification_manager.py

```diff
@@ -45,7 +45,7 @@ async def create(
     async def potentially_add_new_notification(self, coin_state: CoinState, parent_spend: CoinSpend) -> bool:
         if (
             coin_state.spent_height is None
-            or not self.wallet_state_manager.wallet_node.config.get("accept_notifications", False)
+            or not self.wallet_state_manager.wallet_node.config.get("enable_notifications", True)
             or self.wallet_state_manager.wallet_node.config.get("required_notification_amount", 0)
```

Comment:
If we're going to have this on by default, we also need a reasonable filter number so people can't spam us for free.

Reply:
filter number?

Reply:
`"required_notification_amount"`